cli/step_activator.go

```diff
@@ -61,87 +57,25 @@ func (a stepActivator) activateStep(
 		stepYMLPth = activatedStep.StepYMLPath
 		origStepYMLPth = activatedStep.OrigStepYMLPath
 	} else if stepIDData.SteplibSource != "" {
-		stepInfo, didUpdate, err := activateStepLibStep(stepIDData, stepDir, stepYMLPth, isStepLibUpdated, isSteplibOfflineMode)
-		didStepLibUpdate = didUpdate
-
-		stepInfoPtr.ID = stepInfo.ID
```

Comment:
Is the `stepInfoPtr` update happening within activator.ActivateSteplibRefStep?

Reply:
yes yes https://github.com/bitrise-io/stepman/blob/master/activator/steplib_ref.go#L43